src/include/kdbplugin.h

```diff
@@ -108,7 +108,8 @@ Plugin * elektraPluginExport (const char * pluginName, ...);
 KeySet * elektraPluginGetConfig (Plugin * handle);
 void elektraPluginSetData (Plugin * plugin, void * handle);
 void * elektraPluginGetData (Plugin * plugin);
-
+void elektraPluginSetGlobal (Plugin * plugin, KeySet * ks);
```

Comment:
Is this a correct name? If it is only available via a single invocation it is not exactly global, is it?

Maybe elektraPluginSetSharedData?

(And elektraPluginSetGlobalData for actually globally available data within a KDB handle?)

Reply:
You're right, this could be misleading terminology. Maybe I can come up with something better.

For my usecase, I only need what I described in `doc/decisions/global_plugins.md`.

Reply:
Maybe elektraPluginGetGlobalDataOfCurrentInvocation fits better? (and elektraPluginGetGlobalData if it is really global)

Reply:
I thought about what makes sense and reworked it to be global within a KDB handle and called it `elektraPluginGetGlobalKeySet`.